content-learn/mastering-meshery-consul/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Mastering Meshery and Consul'
+description: 'Learn how to build your own addons that will super charge your development'
+heroDescription: 'Addons give you superpowers to extend Storybook, automate workflows, and integrate with your favorite tools. This guide shows you how to create an addon.'
+overview: 'While the Storybook community offers over 250 addons, you can also build one tailored to your specific needs. This guide introduces you to the Addon Kit and APIs, along the way you will build an addon from scratch.'
+order: 4
+themeColor: '#EBC017'
+heroAnimationName: null,
+cardImage: '../../src/assets/images/service-mesh-icons/linkerd-white.svg'
+status: 'new'
+learningPaths: 4
```

Comment:
This is not to be added as a frontmatter but fetched dynamically using graphql. Though, you can ignore this in the update. It will be handled in a future iteration.